### ISSUE TITLE
Cursor Position Bugged

### ISSUE THREAD

**Issue by ienooo** [X9VoiD/VoiDPlugins#34]
Mode: Windows Ink Relative Mode
Tablet: XP-Pen Star G640

When using my tablet, the cursor teleports in the upper left corner of my screen and I'm only able to use it in that small area (~[0,0]-[330,30]).
When I move my mouse again the cursor teleports back to the position where it was before I used my tablet.

Running a python script that returns the mouse position always returns the last position where the mouse was not where the cursor is while I use my tablet. 
So when I moved the cursor with my tablet from eg [500,500] to [0,0] it returns [500,500] anyways. Moving my mouse teleports the cursor back to [500,500].
The script i used:
```py
from ctypes import windll, Structure, c_long, byref

class POINT(Structure):
    _fields_ = [("x", c_long), ("y", c_long)]


def queryMousePosition():
    pt = POINT()
    windll.user32.GetCursorPos(byref(pt))
    return { "x": pt.x, "y": pt.y}

pos = queryMousePosition()
print(pos)
```

**Comment by jamesbt365** [comment]
Try the latest release, if it still happens its OpenTabletDriver/OpenTabletDriver/issues/1124

**Comment by X9VoiD** [comment]
Resolved by 37f047198276a9b280c1732f056a0ed44cd73eca
